.env.example

```diff
@@ -9,3 +9,4 @@ REDIS_URL=
 SENTRY_DSN=
 SENTRY_TRACE_SAMPLE_RATE=
 NEYNAR_API_KEY=
+SN_INFURA_API_KEY=
```

Comment:
Can we rename to this:
```suggestion
INFURA_API_KEY=
```

Reply:
I've added already the key in Stamp server (with and without SN_ prefix)

Reply:
Can you also add it to github so tests pass?

Reply:
Should be ok now
<img width="959" alt="image" src="https://github.com/snapshot-labs/stamp/assets/16245250/05cb50ab-5475-417a-a6e4-a09906bb79d7">